File: src/main/java/com/dh/ondot/member/core/exception/AppleSignatureInvalidException.java
package com.dh.ondot.member.core.exception;

import com.dh.ondot.core.exception.UnauthorizedException;

import static com.dh.ondot.core.exception.ErrorCode.APPLE_SIGNATURE_INVALID;

public class AppleSignatureInvalidException extends UnauthorizedException {
    public AppleSignatureInvalidException() {
        super(APPLE_SIGNATURE_INVALID.getMessage());
    }

    @Override
    public String getErrorCode() {
        return APPLE_SIGNATURE_INVALID.name();
    }
}

File: src/main/kotlin/com/dh/ondot/member/core/exception/AppleSignatureInvalidException.kt
package com.dh.ondot.member.core.exception

import com.dh.ondot.core.exception.ErrorCode.APPLE_SIGNATURE_INVALID
import com.dh.ondot.core.exception.UnauthorizedException

class AppleSignatureInvalidException :
    UnauthorizedException(APPLE_SIGNATURE_INVALID.message) {
    override val errorCode: String get() = APPLE_SIGNATURE_INVALID.name
}
